apps/debt/views/debt_export_view.py

```
@@ -0,0 +1,66 @@
+
+from django.http import StreamingHttpResponse
+from django.utils import timezone
+from django.views import View
+
+from apps.debt.models import Debt
+from apps.debt.views.mixins.debt_base_context import DebtBaseContext
+from apps.room.views.mixins import RoomMembershipRequiredMixin
+
+
+class DebtExportView(RoomMembershipRequiredMixin, DebtBaseContext, View):
+    HEADER = ["debitor", "creditor", "amount", "currency", "settled", "settled_at"]
```

Ruff (0.14.14)

[warning] Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)